eventuate-core/src/main/scala/com/rbmhtechnology/eventuate/EventsourcedView.scala

```diff
@@ -417,10 +426,19 @@ trait EventsourcedView extends Actor with Stash {
       events.foreach(receiveEvent)
       replay(progress + 1L)
     }
-    case ReplayFailure(cause, iid) => if (iid == instanceId) {
-      logger.error(cause, s"replay failed, stopping self")
-      Try(onRecovery(Failure(cause)))
-      context.stop(self)
+    case ReplayFailure(cause, from, iid) => if (iid == instanceId) {
+      val remainingAttempts = replayAttempts - 1
```

Comment:
Is `remainingAttemps` really needed? Set `replayAttempts` parameter accordingly and use it directly.